### PR TITLE
Feature cluster

### PR DESCRIPTION
**Pull request by jkrol2**

### CODE REVIEW

File: .vagrant/machines/ansible-elasticsearch/virtualbox/vagrant_cwd [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/kuba/ansible-elasticsearch
```
Comment (pkrupa2) [inline .vagrant/machines/ansible-elasticsearch/virtualbox/vagrant_cwd]: This file is not needed. Please delete it.

File: molecule.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
+
```
Comment (pkrupa2) [inline molecule.yml]: Redundant line

File: playbook.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (pkrupa2) [inline playbook.yml]: This file should be located in `tests` directory.

File: defaults/main.yml [hunk 4]
```diff
@@ -23,7 +23,10 @@ elasticsearch_install_dir: "{{ ansible_user_dir }}/elasticsearch"
 
 # Configuration
 elasticsearch_instance_name: "fromage"
-elasticsearch_config: {}
+
+#Cluster
+elasticsearch_cluster_name: "Elasticsearch Cluster"
```
Comment (pkrupa2) [inline defaults/main.yml]: Where is this variable used?

File: defaults/main.yml [hunk 5]
```diff
@@ -23,7 +23,10 @@ elasticsearch_install_dir: "{{ ansible_user_dir }}/elasticsearch"
 
 # Configuration
 elasticsearch_instance_name: "fromage"
```
Comment (pkrupa2) [inline defaults/main.yml]: Where is this variable used?

File: templates/elasticsearch.yml.j2 [hunk 6]
```diff
@@ -1,20 +1,4 @@
 # {{ ansible_managed }}
-{% if elasticsearch_config %}
```
Comment (pkrupa2) [inline templates/elasticsearch.yml.j2]: Why those lines are deleted?

Comment (jkrol2) [reply templates/elasticsearch.yml.j2]: They are not used

File: tests/inventory [hunk 7]
```diff
@@ -1 +1,4 @@
-localhost
+[elasticsearch]
```
Comment (pkrupa2) [inline tests/inventory]: File not needed.

**Pull request merged**